src/map/mob.cpp

```diff
@@ -1858,7 +1858,14 @@ static bool mob_ai_sub_hard(struct mob_data *md, t_tick tick)
 
 	if ((mode&MD_AGGRESSIVE && (!tbl || slave_lost_target)) || md->state.skillstate == MSS_FOLLOW)
 	{
+		int prev_id = md->target_id;
```

Comment:
```suggestion
		int32 prev_id = md->target_id;
```

Reply:
md->target_id is of type "int", do you really want to add an implicit cast here?